src/kbmod/fake_data_creator.py

```diff
@@ -0,0 +1,206 @@
+import os
+import sys
+import random
+
+from astropy.io import fits
+from pathlib import Path
+
+from kbmod.search import *
+
+
+class FakeDataSet:
+    """This class creates fake data sets for testing and demo notebooks."""
+
+    def __init__(self, width, height, num_times, noise_level=2.0, use_seed=False):
+        """The constructor.
+
+        Parameters
+        ----------
+        width : int
+            The width of the images in pixels.
+        height : int
+            The height of the images in pixels.
+        num_times : int
+            The number of times steps (number of images).
+        noise_level : float
+            The level of the background noise.
+        use_seed : bool
+            Use a deterministic seed to avoid flaky tests.
+        """
+        self.width = width
+        self.height = height
+        self.psf_val = 0.5
+        self.noise_level = noise_level
+        self.trajectories = []
+
+        # Generate times with multiple observations per night
+        # separated by ~15 minutes.
+        self.times = []
+        seen_on_day = 0
+        day_num = 0
+        for i in range(num_times):
+            t = 57130.2 + day_num + seen_on_day * 0.01
+            self.times.append(t)
+
+            seen_on_day += 1
+            if seen_on_day == 3:
+                seen_on_day = 0
+                day_num += 1
+
+        # Make the image stack.
+        self.stack = self.make_fake_image_stack(use_seed)
+
+    def make_fake_image_stack(self, use_seed=False):
+        """Make a stack of fake layered images.
+
+        Returns
+        -------
+        stack : image_stack
+        """
+        img_count = len(self.times)
+        t0 = self.times[0]
+        p = psf(self.psf_val)
```

Comment:
This is just always 0.5. Should it be configurable, too?

Reply:
I made it configurable.